app/controllers/sessions_controller.rb

```diff
@@ -14,8 +14,8 @@ def create
       flash[:success] = {color: 'green', message: "Logged in as #{@user.first_name}"}
       redirect_to dashboard_path
     else
-      flash[:error] = {color: 'orange', message: "Username or Password incorrect."}
-      render :new
+      flash[:error] = {color: 'orange', message: "Username or password incorrect."}
+      redirect_to '/login'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.